app/assets/stylesheets/custom-bootstrap/_variables.scss

```diff
@@ -1,5 +1,10 @@
+// colors settings
+$brand-primary: darken(#f89090, 6.5%);
```

Comment:
Color literals like `#f89090` should only be used in variable declarations; they should be referred to via variable everywhere else.